web3/providers/eth_tester/defaults.py

```diff
@@ -23,46 +36,61 @@
     excepts,
 )
 
+from web3.types import (
+    LogReceipt,
+    RPCResponse,
+    TxReceipt,
+)
+
+if TYPE_CHECKING:
+    from eth_tester import (  # noqa: F401
+        EthereumTester,
+    )
+
 
-def not_implemented(*args, **kwargs):
+def not_implemented(*args: Any, **kwargs: Any) -> NoReturn:
     raise NotImplementedError("RPC method not implemented")
 
 
 @curry
-def call_eth_tester(fn_name, eth_tester, fn_args, fn_kwargs=None):
+def call_eth_tester(
+    fn_name: str, eth_tester: "EthereumTester", fn_args: Any, fn_kwargs: Any=None
+) -> RPCResponse:
     if fn_kwargs is None:
         fn_kwargs = {}
     return getattr(eth_tester, fn_name)(*fn_args, **fn_kwargs)
 
 
-def without_eth_tester(fn):
+def without_eth_tester(fn: Callable[..., Any]) -> Callable[..., RPCResponse]:
```

Comment:
I think this would be more accurately typed as:

```python
TReturn = TypeVar("TReturn")
TParams = TypeVar("TParams")

def without_eth_tester(fn: Callable[[TParams], TReturn]) -> Callable[[EThereumTester, TParams], TReturn]:
    ...
```